helix-view/src/view.rs

```diff
@@ -65,6 +65,9 @@ pub struct View {
     pub jumps: JumpList,
     /// the last accessed file before the current one
     pub last_accessed_doc: Option<DocumentId>,
+    /// the last modified file before the current one
+    pub last_modified_doc1: Option<DocumentId>,
+    pub last_modified_doc2: Option<DocumentId>,
```

Comment:
Why not `last_modified_docs: [Option<DocumentId>; 2]`?

I also don't see an explanation on why we need two fields

Reply:
Sorry, didn't put a comment on that. Yeah, this is good as well.